util.py

```diff
@@ -70,7 +73,15 @@ def split_bbox(output_dir_: str, bbox: str, to_bbox_str: Callable[[float, float,
             prev_lat = min_lat
             while prev_lat < max_lat:
                 cur_lat = min(prev_lat + section_size, max_lat)
-                bbox_sections.append(to_bbox_str(prev_long, prev_lat, cur_long, cur_lat))
+
+                # Convert the long / lat bbox bounds to a string that the trace source API can understand (using the
+                # given lambda)
+                bbox_str = to_bbox_str(prev_long, prev_lat, cur_long, cur_lat)
+
+                # The file on disk where we will store trace data
+                result_filename = os.path.join(output_dir_, bbox + '.pickle')
+
+                bbox_sections.append((bbox_str, result_filename))
```

Comment:
Small edit here to return both the string representation of the bbox section as well as the filename / path that we should write the API results to.